cegs_portal/search/templates/search/v1/partials/_multi_experiment_index.html

```diff
@@ -34,23 +34,147 @@
                     {% endfor %}
                 </div>
             </div>
-            <div class="container" id="selected-experiments">
-                <div class="flex justify-center items-center">
-                    <div class="font-bold text-2xl text-slate-500">＋</div>
-                    <div class="text-xl font-bold text-slate-500">Selected Experiments</div>
+            <div class="container">
+                <div
+                  class="rounded-t-lg bg-white dark:bg-body-dark">
+                  <h2 class="mb-0" id="headingOne">
+                    <button
+                      class="group relative flex w-full items-center px-5 py-4 text-left text-base text-neutral-800 transition [overflow-anchor:none] hover:z-[2] focus:z-[3] focus:outline-none dark:bg-body-dark dark:text-white [&:not([data-twe-collapse-collapsed])]:bg-white [&:not([data-twe-collapse-collapsed])]:text-primary [&:not([data-twe-collapse-collapsed])]:shadow-border-b dark:[&:not([data-twe-collapse-collapsed])]:bg-surface-dark dark:[&:not([data-twe-collapse-collapsed])]:text-primary dark:[&:not([data-twe-collapse-collapsed])]:shadow-white/10 "
+                      type="button"
+                      data-twe-collapse-init
+                      data-twe-target="#collapseOne"
+                      aria-expanded="true"
+                      aria-controls="collapseOne">
+                      <div class="font-bold text-2xl text-slate-500 mb-2">＋</div>
+                      <div>
+                        <div class="text-xl font-bold text-slate-500">Select Experiments</div>
+                      </div>
+                      <span
+                        class="-me-1 ms-auto h-5 w-5 shrink-0 rotate-[-180deg] transition-transform duration-200 ease-in-out group-data-[twe-collapse-collapsed]:me-0 group-data-[twe-collapse-collapsed]:rotate-0 motion-reduce:transition-none [&>svg]:h-6 [&>svg]:w-6">
+                        <svg
+                          xmlns="http://www.w3.org/2000/svg"
+                          fill="none"
+                          viewBox="0 0 24 24"
+                          stroke-width="1.5"
+                          stroke="currentColor">
+                          <path
+                            stroke-linecap="round"
+                            stroke-linejoin="round"
+                            d="M19.5 8.25l-7.5 7.5-7.5-7.5" />
+                        </svg>
+                      </span>
+                    </button>
+                  </h2>
+                  <div
+                    id="collapseOne"
+                    class="!visible"
+                    data-twe-collapse-item
+                    data-twe-collapse-show
+                    aria-labelledby="headingOne"
+                    data-twe-parent="#accordionExample">
```

Comment:
I don't see anything with an id of "accordionExample", but also you should pick a different id.

Note: This is done in multiple places on both pages, so please update them all.